webapp/storybook/config.js

```diff
@@ -22,6 +22,20 @@ Vue.component('v-popover', {
   template: '<div><slot>Popover Content</slot></div>',
 })
 
+// Setup design token addon
+const scssReq = require.context('!!raw-loader!~/view/styles', true, /.\.scss$/)
+const scssTokenFiles = scssReq
+  .keys()
+  .map(filename => ({ filename, content: scssReq(filename).default }))
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-07T11:04:40Z" title="Thursday, November 7th 2019, 12:04:40 pm +01:00">Nov 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -22,6 +22,20 @@ Vue.component('v-popover', {
   template: '<div><slot>Popover Content</slot></div>',
 })
 
+// Setup design token addon
+const scssReq = require.context('!!raw-loader!~/view/styles', true, /.\.scss$/)
+const scssTokenFiles = scssReq
+  .keys()
+  .map(filename => ({ filename, content: scssReq(filename).default }))
```


That's quite interesting. I wonder in which places we could re-use that. Did you copy this from somehwere @alina-beck?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-11T07:04:40Z" title="Monday, November 11th 2019, 8:04:40 am +01:00">Nov 11, 2019</time>_

---


Yep, this came from the [design token addon](https://github.com/UX-and-I/storybook-design-token) for storybook.

I don't think it's possible to change token values _in_ the storybook interface but you should be able to change them in the `.scss` file and see the change reflected in storybook immediately.